### PR TITLE
[RPA-694] Official LP texts

### PR DESCRIPTION
**Pull request by LukaszKaczkowski**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const Navbar: FC<NavbarProps> = ({
                         {translate('Landing.Header.Nav.Option.RPA')}
                     </Link>
                 </li>
-                <li className={styles.listItem}>
+                {/* <li className={styles.listItem}>
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx:57]: almost never leave commented code

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/MainView/index.tsx [hunk 2]
```diff
@@ -18,9 +18,9 @@ const MainView = () => (
         <BenefitsSection />
         <RPASection />
         <ProsSection />
-        <IndustriesSection />
+        {/* <IndustriesSection /> */}
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/MainView/index.tsx:21]: are this comments intentional?

**Review by star-in-the-jar — DISMISSED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (preludium) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/components/Navbar/Navbar.tsx:57]: this is acceptable, we want to hide part of the LP, thus header needs adjustment too

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/MainView/index.tsx (hunk 2 above)
Comment (preludium) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/MainView/index.tsx:21]: yup

**Review by preludium — DISMISSED** [review]

**Review by preludium — APPROVED** [review]

**Pull request merged**